code/addons/docs/template/stories/docs2/Canvas.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
```

Comment:
Instead of this, could you add a story to https://github.com/storybookjs/storybook/blob/next/code/ui/blocks/src/blocks/internal/InternalCanvas.stories.tsx instead? That's where we have stories for deprecated APIs for the Canvas block.